From b18ba5f646239725f3c3e3406e34b75e7287c495 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Thu, 9 Apr 2015 16:19:35 +0800
Subject: [PATCH 0543/5242] MLK-10629-3 dts: imx7d-sdb: add wifi-host property
 for usdhc2

commit  13af9aae5b97b54af173bfce149c963b49667dfb from
https://source.codeaurora.org/external/imx/linux-imx.git

There's a Broadcom WiFi on usdhc2.
Add this property due to Broadcom WiFi driver needs call wifi_card_detect
function.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
(cherry picked from commit 72015985537b5bd14ac7b0c97860ab26309ad1a1)
Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 53a89fe..7696a1d 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -707,6 +707,7 @@
 	non-removable;
 	vmmc-supply = <&reg_brcm>;
 	fsl,tuning-step = <2>;
+	wifi-host;
 	status = "okay";
 };
 
-- 
1.7.9.5

